### DISCUSSION TITLE
How to import Sqlite into email client?

### DISCUSSION THREAD

**Discussion by NAL100** [GAM-team/got-your-back#377] [Q&A]
I was able to complete a backup of an email account, but now have a bunch of .eml files and an .sqlite which I'm not sure what to do with. What can I do with this in order to browse these files with an email client?

**Comment by jay0lee** [comment]
GYB doesn't support import into other email clients. The propietary backup
format is meant to be restored (by GYB) to Gmail.

If you want to backup Gmail and restore to another email client, use
Google's Takeout feature.

Jay
